shared-libs/cht-datasource/src/person.ts

```diff
@@ -87,6 +87,35 @@ export namespace v1 {
       return curriedFn;
     };
 
+  const getPeopleGenerator = () => (context: DataContext) => {
```

Comment:
I think we can just combine this function with `getAll`, since I don't think it is going to get reused anywhere...